app/assets/stylesheets/pin-index.css.scss

```diff
@@ -0,0 +1,130 @@
+$primary-background-color: rgba(237, 242, 242, 0.8);
+
+
+#container{
+  background-color: $primary-background-color;
+}
+
+hr{
```

Comment:
This is maybe not specific enough, unless you want to change all hr in your app?